docs/source/user-guide/dataset_object.rst

```diff
@@ -83,46 +81,57 @@ whether the column is categorical or not:
 * else:
     *  number of unique < `max_categories` AND (number of unique / number of samples) < `max_categorical_ratio`
 
-Creating a Dataset
-======================
+Useful Functions
+================
+
+Create Dataset From a Numpy Arrays
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A Dataset can be created using a 2D numpy array for features and 1D numpy array for the labels. The features array is mandatory, and the labels array is optional.
+
+    >>> features = np.array([[0.25, 0.3, 0.3], [0.14, 0.75, 0.3], [0.23, 0.39, 0.1]])
+    >>> labels = np.array([0.1, 0.1, 0.7])
+    >>> dataset_with_labels = Dataset.from_numpy(features, labels)
+    >>> dataset_without_labels = Dataset.from_numpy(features)
+
+Also, it's possible to assign names to the features and label:
+
+    >>> Dataset.from_numpy(
+    ...     features, labels,
+    ...     feature_names=['feat1', 'feat2', 'feat3',],
+    ...     label_name='target'
+    ... )
+
+All the rest of the Dataset's properties can be passed also as a regular keyword arguments:
+
+    >>> Dataset.from_numpy(
+    ...     features, labels,
+    ...     feature_names=['feat1', 'feat2', 'feat3',],
+    ...     label_name='target',
+    ...     max_float_categories=10
+    ... )
```

Comment:
```suggestion
All the rest of the Dataset's properties can be passed also as regular keyword arguments:

    >>> Dataset.from_numpy(
    ...     features, labels,
    ...     feature_names=['feat1', 'feat2', 'feat3',],
    ...     label_name='target',
    ...     max_float_categories=10
    ... )
```

Suggested a small grammar fix, but not sure - why is this needed - what does it demonstrate?

Reply:
Did you run this code example? I think this doesn't work since the features names won't match the column names